backend/src/main.cpp

```diff
@@ -19,17 +20,32 @@ int main()
     db.execute("INSERT INTO passwords (password) VALUES ('chocolate1');");
     db.printTable("passwords");
 
-    // declare crow application
-    crow::SimpleApp app;
+    // Enable CORS
+    crow::App<crow::CORSHandler> app;
+
+    // Customize CORS
+    auto &cors = app.get_middleware<crow::CORSHandler>();
+
+    // clang-format off
+    cors
+      .global()
+        .headers("*")
+        .methods("POST"_method, "GET"_method)
+      .prefix("/nocors")
+        .ignore();
+    // clang-format on
 
-    // define endpoint at the root directory
     CROW_ROUTE(app, "/")
     ([]()
      {
         crow::json::wvalue response;
         response["status"] = "success";
         return response; });
 
+    CROW_ROUTE(app, "/cors")
+    ([]()
+     { return "Check Access-Control-Allow-Origin header"; });
+
```

Comment:
**thought**: This endpoint should not exist after this sprint. I'm fine with it staying for now while we make the future issues work, but we should remember to get rid of it.

Reply:
Agreed, I think it may be beneficial for now just to ensure that the backend continues to work with a simple endpoint ("/cors"). Next endpoint I will likely end up changing this when computing set intersection, since then the simple endpoint will just be "/".